<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>309</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>309</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control Panel</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QToolButton" name="button_open_result">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Open Measurement Results</string>
      </property>
      <property name="whatsThis">
       <string>Open Measurement Results</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/open_results.png</normaloff>icons/open_results.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="button_measurement">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Droplet Measurement</string>
      </property>
      <property name="whatsThis">
       <string>Droplet Measurement</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/measurement.png</normaloff>icons/measurement.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="button_side_cam">
      <property name="toolTip">
       <string>Side Camera</string>
      </property>
      <property name="whatsThis">
       <string>Side Camera</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/side_cam_off.png</normaloff>icons/side_cam_off.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="button_top_cam">
      <property name="toolTip">
       <string>Top Camera</string>
      </property>
      <property name="whatsThis">
       <string>Top Camera</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/top_cam_off.png</normaloff>icons/top_cam_off.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="button_setting">
      <property name="toolTip">
       <string>Settings</string>
      </property>
      <property name="whatsThis">
       <string>Settings</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/settings.png</normaloff>icons/settings.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
